### ISSUE TITLE
Entity modals are not displayed in Firefox for Android (works in Chrome)

### ISSUE THREAD

**Issue by synthead** [home-assistant/frontend#7851]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

When tapping on a light entity on my desktop browser, this modal appeara:

![image](https://user-images.githubusercontent.com/820984/100653802-c8afa100-32fd-11eb-8d2e-3a9fceff2e3e.png)

However, in Firefox for Android, this modal is not displayed when the entity is tapped.

This functionality works correctly on Chrome for Android.

## Expected behavior

Modals appear when light entities are tapped in Firefox for Android.

## Steps to reproduce

1. Install Firefox for Android on an Android device.
2. Log into HA using Firefox for Android.
3. Tap on a light entity.
4. Observe that the expected modal does not appear.

## Environment

- Home Assistant release with the issue: 0.117.5
- Last working Home Assistant release (if known): This used to work 1-2 months ago (sorry for being vague)
- Browser and browser version: Firefox for Android
- Operating system: Arch Linux (server), Android (client)

## State of relevant entities

```yaml
integration_id: 4
area_name: Stairs
friendly_name: Stair light
icon: 'mdi:stairs'
supported_features: 33
```

## Problem-relevant configuration

_The stair light entity was added with an integration and there is no YAML for it, but this issue affects every entity that would display a modal_

## Javascript errors shown in your browser console/inspector

_No errors_

**Comment by bramkragten** [comment]
What is de lovelace configuration, or what card/place are you tapping the light entity?

**Comment by synthead** [comment]
Here is a card that has this issue:

```yaml
entities:
  - entity: light.basement_lights
title: Basement
type: entities
```

This is how this card looks in the mobile UI:
![image](https://user-images.githubusercontent.com/820984/101711205-f906e600-3a47-11eb-96ef-d80cf27303c5.png)

I am tapping on the "Basement lights" text in an attempt to view its modal.

**Comment by synthead** [comment]
This seems to be fixed in 2020.12.0-1. Closing issue.
